assets/src/styles/blocks/Covers/TakeActionCovers.scss

```diff
@@ -154,12 +154,12 @@
       display: block;
 
       --block-covers--card-btn-- {
```

Comment:
All styles for the covers button are inside of the `:focus, ...` selector of the card. Probably we only need to put the `display` property in here so that other styles can be regardless of the hover.

Reply:
I didn't get what you meant by 
> Probably we only need to put the display property in the pseudo selector so that other styles can be regardless of the hover.

Can you explain with images or code ? `.btn` has a different background and border color on hover, I don't get if/how you want to regroup styles there :thinking:

Reply:
Yeah in this case both code and images are more explanatory :ok_hand: https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/572 Also noticed that I used a different name in the one on the bottom. We could include these changes, however it adds some risk by changing CSS structure.

Currently if you remove the `display: none` rule, that hides the button when not hovered, it looks like this:
![Screenshot from 2021-04-28 09-56-36](https://user-images.githubusercontent.com/7604138/116369943-378c1a00-a80a-11eb-9f35-4e1a8a794e5d.png)

Reply:
See https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/523/files#r622006056